src/main/java/com/hbm/items/special/ItemBedrockOreNew.java

```diff
@@ -169,6 +169,11 @@ public static ItemStack extract(Object o, int amount) {
 			List<ItemStack> billets = OreDictionary.getOres(frame.billet(), false); if(!billets.isEmpty()) return fromList(billets, amount);
 			List<ItemStack> ingots = OreDictionary.getOres(frame.ingot(), false); if(!ingots.isEmpty()) return fromList(ingots, amount);
 		}
+		if(o instanceof ItemStack){
```

Comment:
At the time when the enum is initiated, most items don't even exist yet, so using items or item stacks doesn't make a whole lot of sense.

Reply:
I only used that on my recipe maker, so it can handle any kind of item byproduct including crumbs